diff --git i/include/date/date.h w/include/date/date.h
index 79ab0da..64be58f 100644
--- i/include/date/date.h
+++ w/include/date/date.h
@@ -32,11 +32,7 @@
 // We did not mean to shout.
 
 #ifndef HAS_STRING_VIEW
-#  if __cplusplus >= 201703
-#    define HAS_STRING_VIEW 1
-#  else
-#    define HAS_STRING_VIEW 0
-#  endif
+#  define HAS_STRING_VIEW 1
 #endif  // HAS_STRING_VIEW
 
 #include <cassert>
